# Response parameters:
float32 cost_experiment 									# Cost value